### ISSUE TITLE
How to build the app

### ISSUE THREAD

**Issue by mariecl** [realmacsoftware/RMConnecter#29]
Hey,

Sorry for the dumb question. Would you mind giving some more instructions regarding how to actually build Connecter?
Thanks a lot.
Best,

  Marie.

**Comment by nikf** [comment]
Hi Marie, Connecter is an OS X app, so the latest Xcode toolchain (and a code-signing identity) should do the trick.

**Comment by mariecl** [comment]
Yes, I noticed from the code, but other people might not? It might be worth adding it somewhere in the README so that people with no Apple developers account would not try to use the app.

**Comment by nikf** [comment]
Gotcha! Will get it clarified!

**Comment by mariecl** [comment]
OK, great! Thanks :)

**Comment by services-thirdparty** [comment]
Hey, guys! We announced pre-order for  brand new application, that can help you upload any metadata to App Store Connect (localizations, screenshots, in-app purchases, etc.) We're not sure that our product really need for community, but we're want to continue developing. Please, if you're have some disadvantage with App Store Connect, support us:

https://apps.apple.com/app/metadata-connect/id1494035456?mt=12
